src/repositories/bookmarks.repository.ts

```diff
@@ -0,0 +1,23 @@
+import { ObjectId } from "mongodb";
+
+import { BookmarkModel } from "../models/Bookmark";
+
+export const listBookmarks = async () => {
+  const result = await BookmarkModel.find();
+  return result;
+};
+
+export const addOne = async (tweetId: ObjectId, ownerId: ObjectId) => {
+  const result = new BookmarkModel({
```

Comment:
In that place we need to string-type parameters of function. And inside function take ObjectId by string. Only repository has to know how works with database.